docs/guides/creatingasource.md

```diff
@@ -0,0 +1,118 @@
+# Creating a Source
+
+In this guide we will create a point to point bridge between an AWS SQS queue and a microservice application called `sockeye` which displays events in web interface. This simple flow is depicted below.
+
+![](../assets/images/sqs-sockeye.png)
+
+We will create:
+
+- [x] The sockeye target which serves as an event display.
+- [x] The `AWSSQSSource` which consumes events from an AWS SQS queue.
+
+## Event display
+
+First of all, we need to have a tool to see the events that come from our source.
+
+Create a `sockeye` service by saving the following YAML manifest in a file called `sockeye.yaml` and applying it to your Kubernetes cluster:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: sockeye
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
+```
+
+```
+kubectl apply -f sockeye.yaml
+```
+
+Open the web interface in a browser:
+
+```shell
+browse $(kubectl get ksvc sockeye -o=jsonpath='{.status.url}')
```

Comment:
(Thanks for using plain JSONpath in this one!)

`browse` is a Mac specific command though. I think we are better off just printing the URL and letting users do what they want with it. Besides, most terminal emulators support Ctrl+click on URLs.